getting-started/kubernetes/vpp/baremetal.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Install VPP on a baremetal cluster
+description: Install Calico with the VPP dataplane on a baremetal cluster
+canonical_url: '/getting-started/kubernetes/vpp/baremetal'
```

Comment:
The template headings are:

Big Picture
Value
Before You Begin (aka Requirements)
Concepts
How To
  - Inline link
  - Inline link
Next Steps or Above and beyond